recipe/meta.yaml

```diff
@@ -24,20 +22,30 @@ requirements:
     - setuptools
     - wheel
     - python
-    - leveldb
+    - leveldb 1.23
   run:
     - python
     - leveldb
```

Comment:
I think `leveldb` in `run` should be `leveldb >=1.23,<1.24.0a0`, see https://metayaml-conda.fly.dev/metayaml/reverse_dependencies?name=leveldb